releasenotes/notes/hyperbolic-random-graph-d85c115930d8ac08.yaml

```diff
@@ -0,0 +1,11 @@
+features:
+  - |
+    Adds new random graph generator function, :func:`.undirected_hyperbolic_random_graph`
+    to generate a random graph using the H² random graph model.
+  - |
+    Adds new function to the rustworkx-core module ``rustworkx_core::generators``
+    ``hyperbolic_random_graph()`` that generates a H^2 hyperbolic random graph.
+
+issues:
+  - |
+    Related to issue `#150 <https://github.com/Qiskit/rustworkx/issues/150>`.
```

Comment:
The `issues` release note is related to known issues at release time, not Github issues. You can ommit this, the `features` section is all you need